Gopkg.toml

```diff
@@ -41,13 +41,13 @@ required = [
   version = "=0.9.1"
 [[constraint]]
   name = "github.com/golang/protobuf"
```

Comment:
Let's add "google.golang.org/protobuf" to this list of key deps

Reply:
Added.